packages/interactivity/src/utils.ts

```diff
@@ -48,12 +56,18 @@ const afterNextFrame = ( callback: () => void ) => {
  *
  * @return Promise
  */
-export const splitTask = () => {
-	return new Promise( ( resolve ) => {
-		// TODO: Use scheduler.yield() when available.
-		setTimeout( resolve, 0 );
-	} );
-};
+export const splitTask =
+	'scheduler' in window &&
```

Comment:
The checks here can probably be simplified now that the `scheduler` is being defined on the `Window` interface above.

Reply:
Done in 0abf7be